fast-file-mover/src/main/java/com/apeshko/fastfilemover/services/FastFileMover.java

```diff
@@ -0,0 +1,8 @@
+package com.apeshko.fastfilemover.services;
+
+import java.io.FileNotFoundException;
+import java.io.IOException;
+
+public interface FastFileMover {
+    void move(String filePath, String nextFilePath) throws IOException;
```

Comment:
nextFilaPath seems a bit strange, not quite clear what it means, perhaps smth like this?
`void move(String sourceFilePath, String destinationFilePath) throws IOException;`

Reply:
@mikalai-amusin agree, thanks!